modules/op-installing-sbo-operator-using-the-web-console.adoc

```diff
@@ -20,8 +20,9 @@ You have access to an {product-title} cluster using an account with `cluster-adm
 . On the *Install Operator* page:
 +
 .. Select *All namespaces on the cluster (default)* for the *Installation Mode*. This mode installs the Operator in the default `openshift-operators` namespace, which enables the Operator to watch and be made available to all namespaces in the cluster.
-
-.. Select *Automatic* for the *Approval Strategy*. This ensures that the future upgrades to the Operator are handled automatically by the Operator Lifecycle Manager (OLM). If you select the *Manual* approval strategy, OLM creates an update request. As a cluster administrator, you must then manually approve the OLM update request to update the Operator to the new version.
++
+// .. Select *Automatic* for the *Approval Strategy*. This ensures that the future upgrades to the Operator are handled automatically by the Operator Lifecycle Manager (OLM). If you select the *Manual* approval strategy, OLM creates an update request. As a cluster administrator, you must then manually approve the OLM update request to update the Operator to the new version.
+// Add this step once OLM descriptors are supported again.
```

Comment:
Do not remove this step. That does not have anything to do with the removed feature of SBO.

This procedure is about how to install SBO using OLM... which is unrelated to what we have removed. That is SBO's ability to read other operators' OLM descriptors to expose binding data.